packages/firestore/test/integration/api/bundle.test.ts

```diff
@@ -252,3 +264,178 @@ apiDescribe('Bundles', (persistence: boolean) => {
     });
   });
 });
+
+apiDescribe('Bundles conformance', (persistence: boolean) => {
+  const CONFORMANCE_DOC = {
```

Comment:
The approach of "hardcoding" the JSON strings from Node and Java has some merits, but since it is Monday morning and we have a full week ahead of us, I would like to point out that you could also use the Web serializer to built both Proto3 JSON and ProtobufJS documents in the tests themselves. This approach has some benefits (less code to copy/paste, less chance that the code will go out of sync) and some downsides (less portable, may not catch bugs in the Node/Java clients) so I will leave the decision to you.

If we do decide to only support Proto3 JSON, we can also get rid of one of the tests here.

Reply:
I added a todo instead, my 3 minute research did not tell me clearly how to do it..

Reply:
We might want to spend some more time on this since adding this and then removing it right away just adds code churn. You should be able to call [wrap()](https://github.com/firebase/firebase-js-sdk/blob/4090271bb71023c3e6587d8bd8315ebf99b3ccd7/packages/firestore/test/util/helpers.ts#L178) with `CONFORMANCE_DOC` and get a JSON encoded Value proto. You may have to add an overload so you can control the `useProto3JSON` flag (and as discussed, this only works in api_internal)

Reply:
As discussed offline, this is harder than I expected, and does not bring much value because we still need to hard code elements that are not documents. Let's not do this for now?